### PR TITLE
Fix name and script not showing from the database

### LINKED ISSUE #107: When editing, Step 2 in the pipeline wizard does not display value from the database
When editing a pipeline, in Step 2, the form does not display the step `name` and `script` values from the database.

A/C:

On step 2 in the add/edit pipeline wizard:
- [x] step name is loaded from the saved value in the database
- [x] step script is loaded from the value in the database

(The "language" field appears to be loaded properly already.)

### PR DESCRIPTION
**Pull request by ronardcaktus**
Addresses the bug preventing name and script from displaying when a step is edited. Also, fixes test that was supposed to be a `get`, but was posting data.

Closes #107

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -275,8 +275,10 @@ etlman/media/
 .ipython/
 project.css
 project.min.css
-vendors.js
-*.min.js
+# vendors.js
```
Comment (tobiasmcnulty) [inline .gitignore]:
```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -275,8 +275,10 @@ etlman/media/
...
+project.min.js
+vendors.min.js
+# *.min.js
```
Comment (tobiasmcnulty) [inline .gitignore]:
```suggestion
```

File: etlman/projects/views.py [hunk 3]
```diff
@@ -200,7 +200,17 @@ def new_step_step2(request, project_id, step_id=None):
             "name": session_pipeline["pipeline-name"] + "_script",
             "script": "def main():\n\t...\nif __name__ == '__main__':\n\tmain()",
         }
-        form_step = StepForm(instance=step, initial=initial)
+        if not step:
+            form_step = StepForm(instance=step, initial=initial)
+        else:
+            form_step = StepForm(
+                instance=step,
+                initial={
```
Comment (tobiasmcnulty) [inline etlman/projects/views.py]: `initial` would [never be used](https://docs.djangoproject.com/en/4.1/ref/forms/api/#initial-form-values) here, if I understand correctly. There might be something else going on, or perhaps the bug did not exist to begin with?

Comment (ronardcaktus) [reply etlman/projects/views.py]: I tested again in `main` and in this branch and the bug does exist 🤔

File: etlman/projects/tests/test_views/test_view_pipeline.py [hunk 4]
```diff
@@ -45,13 +47,12 @@ def test_post_data_to_new_pipeline(self, nonadmin_user, nonadmin_client, project
         )
         # Check session for saved values
         assert (
-            nonadmin_client.session[SessionKeyEnum.PIPELINE.value]["pipeline-name"]
+            # Import func and replace SessionKeyEnum.PIPELINE.value
```
Comment (tobiasmcnulty) [inline etlman/projects/tests/test_views/test_view_pipeline.py]:
```suggestion
```

**Review by tobiasmcnulty — APPROVED** [review]
🐞

**Pull request merged**